src/main/webapp/instructorRecovery.jsp

```diff
@@ -0,0 +1,282 @@
+<%@ page trimDirectiveWhitespaces="true" %>
+<%@ page pageEncoding="UTF-8" %>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="t" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<t:recoveryPage>
```

Comment:
Although this page is currently hardcoded for the sake of producing a preliminary UI, make sure to reuse tags such as `instructorPage.tag` and refactoring code in this `jsp` file to be more modular (see `instructorHome.jsp` in `src/main/webapp/jsp` for an example). Even this mockup could have used several tags to lay out the basic code structure for later implementation.

Reply:
Also, the location of this file - it should probably be with files like `instructorHome.jsp` rather than the help page.

Reply:
Yeah sure. These and also the nav bar issues are just because in week 1 I cannot find a way to link to the recovery page like `instructorHomePage`. Will fix them in the following weeks!